src/pages/main/main.tsx

```diff
@@ -0,0 +1,47 @@
+import CitiesList from '../../components/cities-list/cities-list';
+import CityCard from '../../components/city-card/city-card';
+import Header from '../../components/header/header';
+import SortForm from '../../components/sort-form/sort-form';
+import { CitiesListType } from '../../types/cities-types';
+
+
+type MainPagePropsTypes = {
+  placesCount: number;
+  offerCount: number;
+  cityList: CitiesListType;
+}
+
+export default function MainPage(props : MainPagePropsTypes): JSX.Element {
+
+  const fakeArray = new Array(props.placesCount);
```

Comment:
```suggestion
  const fakeArray = Array.from({ length: props.placesCount}, () => {})
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from